src/components/Dashboard.scss

```diff
@@ -93,3 +95,79 @@
     }
   }
 }
+
+.main-layout__header {
+  .main-header__logo {
+    span {
+      font-size: 0.9rem;
+    }
+  }
+}
+
+@media screen and (max-width: 1201px) {
+  .converter {
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+
+    .converter__info {
+      &__content {
+        &::before {
+          display: none
+        }
+      }
+    }
+
+    .converter__calc {
+      width: 500px;
+    }
+  }
+}
+
+@media screen and (max-width: 576px) {
+
+  .main-layout__header {
+    display: flex;
+    align-items: center;
+
+    .main-header {
+      &__logo {
+        font-size: 1.5rem;
```

Comment:
instead of changing `rem` for each element, try to change `px` font size for `html` element. This should make all components font size smaller, keeping default proportions.